api/openapi.yaml

```diff
@@ -66,35 +66,11 @@ paths:
       description: "get transactions"
       operationId: getTransactions
       parameters:
-        - in: query
-          name: take
-          description: "indicates how many transactions can be returned by each page"
-          schema:
-            type: integer
-            default: 20
-        - in: query
-          name: skip
-          description: "indicates how many transactions will be skipped when paginating"
-          schema:
-            type: integer
-            default: 0
-        - in: query
-          name: query
-          description: "query to search transactions, based on transaction name and description"
-          schema:
-            type: string
-        - in: query
-          name: sortBy
-          description: "indicates the sort field for the transactions"
-          schema:
-            type: string
-            enum: [created, name, last_run]
-        - in: query
-          name: sortDirection
-          description: "indicates the sort direction for the transactions"
-          schema:
-            type: string
-            enum: [asc, desc]
+        - $ref: "./parameters.yaml#/components/parameters/take"
+        - $ref: "./parameters.yaml#/components/parameters/skip"
+        - $ref: "./parameters.yaml#/components/parameters/query"
+        - $ref: "./parameters.yaml#/components/parameters/runnableResourceSortBy"
```

Comment:
Question: is there a way to "override" ref attributes? I believe that we have many similar `sortBy` fields with only the `enum` with different values.